### PR TITLE
docs: Use tabs for meltano.yml vs meltano config or env

### PR DESCRIPTION
**Pull request by mjsqu**

### CODE REVIEW

**Comment by mjsqu** [comment]
Change made for all blocks matching a pattern, putting this here in case I need to reuse it:

```
Manage this extra directly in your \[`meltano.yml` project file\]\(project#meltano-yml-project-file\):((?:.|\n)+?)Alternatively.*((?:.|\n)+?)^```$


Manage this extra:\n{% tabs cli %}\n{% tab cli meltano.yml %}$1{% endtab %}\n{% tab cli config %}$2{% endtab %}\n{% endtabs %}
```

**Review by WillDaSilva — COMMENTED** [review]
Thanks @mjsqu!

Could you please split the "CONFIG" tab into a "CLI" and "ENV VAR" tab?

File: docs/src/_concepts/plugins.md [hunk 1]
```diff
@@ -172,15 +172,18 @@ If the catalog does not seem to take effect, you may need to [validate the capab
...
 
 ```yaml{3}
 extractors:
 - name: tap-gitlab
   catalog: extract/tap-gitlab.catalog.json
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab catalog config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:186]:
```suggestion
{% tab catalog terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 2]
```diff
@@ -213,15 +219,18 @@ It is used as the default value for the [`target-postgres`](https://hub.meltano.
...
 
 ```yaml{3}
 extractors:
 - name: tap-gitlab
   load_schema: gitlab_data
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab load_schema config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:233]:
```suggestion
{% tab load_schema terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 3]
```diff
@@ -315,7 +333,9 @@ This allows you to define a full schema for taps such as [`tap-dynamodb`](https:
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs schema %}
+{% tab schema meltano.yml %}
 
 ```yaml{3-7}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:340]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 4]
```diff
@@ -366,7 +390,9 @@ selection rules are typically specified using [`meltano select`](/reference/comm
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs select %}
+{% tab select meltano.yml %}
 
 ```yaml{3-5}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:397]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 5]
```diff
@@ -376,7 +402,8 @@ extractors:
   - commits.*
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab select config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:406]:
```suggestion
{% tab select terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 6]
```diff
@@ -417,7 +447,9 @@ selection filers are typically specified using [`meltano elt`](/reference/comman
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs select_filter %}
+{% tab select_filter meltano.yml %}
 
 ```yaml{6-7}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:454]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 7]
```diff
@@ -429,7 +461,8 @@ extractors:
   - commits
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab select_filter config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:465]:
```suggestion
{% tab select_filter terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 8]
```diff
@@ -469,15 +505,18 @@ a state file is typically provided using [`meltano elt`](/reference/command-line
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs state %}
+{% tab state meltano.yml %}
 
 ```yaml{3}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:512]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 9]
```diff
@@ -469,15 +505,18 @@ a state file is typically provided using [`meltano elt`](/reference/command-line
...
 extractors:
 - name: tap-gitlab
   state: extract/tap-gitlab.state.json
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab state config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:519]:
```suggestion
{% tab state terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 10]
```diff
@@ -526,15 +568,18 @@ It is used as the default value for `dbt`'s `target` setting, and should therefo
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs dialect %}
+{% tab dialect meltano.yml %}
 
 ```yaml{3}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:575]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 11]
```diff
@@ -526,15 +568,18 @@ It is used as the default value for `dbt`'s `target` setting, and should therefo
...
 loaders:
 - name: target-example-db
   dialect: example-db
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab dialect config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:582]:
```suggestion
{% tab dialect terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 12]
```diff
@@ -576,7 +626,8 @@ loaders:
   target_schema: $MELTANO_LOAD_DESTINATION_SCHEMA # Value of `destination_schema` setting
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab target_schema config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:630]:
```suggestion
{% tab target_schema terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 13]
```diff
@@ -566,7 +614,9 @@ It is used as the default value for `dbt`'s `source_schema` setting.
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs target_schema %}
+{% tab target_schema meltano.yml %}
 
 ```yaml{5}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:621]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 14]
```diff
@@ -632,7 +686,9 @@ It is included in the default value for `dbt`'s `models` setting: `$MELTANO_TRAN
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs package_name %}
+{% tab package_name meltano.yml %}
 
 ```yaml{4}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:693]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 15]
```diff
@@ -641,7 +697,8 @@ transforms:
   package_name: facebook_ads
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab package_name config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:701]:
```suggestion
{% tab package_name terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 16]
```diff
@@ -680,7 +742,8 @@ transforms:
 {% endraw %}
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab vars config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:746]:
```suggestion
{% tab vars terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 17]
```diff
@@ -744,7 +810,9 @@ When a file path's value is `True`, the matching files are considered to be mana
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs update %}
+{% tab update meltano.yml %}
 
 ```yaml{3-4}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:817]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 18]
```diff
@@ -764,7 +832,8 @@ files:
 </pre>
 </div>
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab update config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:836]:
```suggestion
{% tab update terminal %}
```

File: docs/src/_concepts/plugins.md [hunk 19]
```diff
@@ -213,15 +219,18 @@ It is used as the default value for the [`target-postgres`](https://hub.meltano.
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs load_schema %}
+{% tab load_schema meltano.yml %}
 
 ```yaml{3}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:226]:
```suggestion
```yaml
```

File: docs/src/_concepts/plugins.md [hunk 20]
```diff
@@ -172,15 +172,18 @@ If the catalog does not seem to take effect, you may need to [validate the capab
 
 ##### How to use
 
-Manage this extra directly in your [`meltano.yml` project file](project#meltano-yml-project-file):
+Manage this extra:
+{% tabs catalog %}
+{% tab catalog meltano.yml %}
 
 ```yaml{3}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:179]:
```suggestion
```yaml
```

**Review by tayloramurphy — APPROVED** [review]

File: docs/src/_concepts/plugins.md [hunk 21]
```diff
@@ -327,7 +347,8 @@ extractors:
         format: date-time
 ```
 
-Alternatively, manage this extra using [`meltano config`](/reference/command-line-interface#config) or an [environment variable](/guide/configuration#configuring-settings):
+{% endtab %}
+{% tab schema config %}
```
Comment (tayloramurphy) [inline docs/src/_concepts/plugins.md:351]:
```suggestion
{% tab schema terminal %}
```

**Comment by tayloramurphy** [comment]
@sbalnojan what do you think of this change? I like it as it brings the focus to the YAML file.

**Comment by mjsqu** [comment]
Just updated to add @WillDaSilva suggestion of splitting out 'env' into another tab. Also aligned all tab groups with the name `extra_usage` so they should all shift at once throughout the page

**Comment by tayloramurphy** [comment]
@mjsqu I quite like that! I'm going to hit merge :shipit:

**Pull request merged**